### ISSUE TITLE
Auto create expedius user

### ISSUE THREAD

**Issue by countable** [open-osp/open-osp#75]
Look at how oscardoc is created in the Oscar source. Copy that but don't force a password reset.

This will allow us to auto set the expedius password when setting up expedius.

**Comment by ferdinandpolpol** [comment]
is the expedius account still a provider?

**Comment by warrendennis** [comment]
Yes the Expedius (Oscar) account is a provider.  Admin privileges only.

**Comment by ferdinandpolpol** [comment]
@warrendennis I've been looking into the oscar codebase. The oscardoc is inserted in oscardata.sql with password `-51-282443-97-5-9410489-60-1021-45-127-12435464-32`. Is this a result of custom encrypt function that oscar uses from `import org.oscarehr.util.EncryptionUtils;`?

**Comment by warrendennis** [comment]
The password is encrypted.  Not really custom, a standard abstraction.  

Why is this a concern?

**Comment by ferdinandpolpol** [comment]
@warrendennis since we want to create expedius user with generated password, how do we encrypt that password the same way oscar does it? 
Or am I able to insert a `security` user wihout having the password encrypted?

**Comment by warrendennis** [comment]
Unfortunately this is not possible.  We need to look at possibly generating an API key.  A little extra dev work required and is a lower priority for now.
